### DISCUSSION TITLE
Security Issue : No Answer

### DISCUSSION THREAD

**Discussion by x7-7** [dnnsoftware/Dnn.Platform#4683] [General]
I've tried contacting security@dnnsoftware.com multiple times with no answer.
Is there a better way to contact in order to report security vulnerabilities?

**Comment by bdukes** [comment]
No, that's the correct approach.  /cc @mitchelsellers

**Comment by mitchelsellers** [comment]
@x7-7 I just checked the alias, we have no pending tickets and nothing incoming, aside from spam, for the last 14 days.

If you have an urgent need, email me directly msellers@iowacomputergurus.com

**Reply by x7-7** [reply]
This account alias is not related to the email send to security@dnnsoftware.com, the first email has been sent on 13th April, I'll send it again now and mention this alias in the beginning of the email.

**Reply by mitchelsellers** [reply]
I got your message, replying now
